index.js

```diff
@@ -1,3 +1,10 @@
 'use strict'; // eslint-disable-line strict
 
+/**
+ * Catch uncaught exceptions and add timestamp to aid debugging
+ */
+process.on('uncaughtException', err => {
```

Comment:
Do we want to also catch the rejections?

Reply:
It's been a long time but this does catch rejections as well